BidCat/API/BiddingController.cs

```diff
@@ -14,22 +14,80 @@ public class BiddingController : ApiProvider
 			new ApiCommand("GetBidsForUser", args => GetAuction(args.Any() ? int.Parse(args.First()) : throw new ApiError("Auction ID was not specified")).GetBidsForUser(args.Skip(1).Any() ? int.Parse(args.Skip(1).First()) : throw new ApiError("User ID was not specified")).Wait(), new List<ApiParameter> { new ApiParameter("Auction ID"), new ApiParameter("User ID") }, "Gets all bids on every lot for the given user"),
 			new ApiCommand("GetBidsForUsers", args => GetAuction(args.Any() ? int.Parse(args.First()) : throw new ApiError("Auction ID was not specified")).GetBidsForUsers(args.Skip(1).Count() == 1 ? JsonConvert.DeserializeObject<IEnumerable<int>>(args.Skip(1).First()) : args.Skip(1).Any() ? args.Skip(1).Select(int.Parse) : throw new ApiError("User ID was not specified")).Wait(), new List<ApiParameter> { new ApiParameter("Auction ID"), new ApiParameter("User IDs", "IEnumerable<int>") }, "Gets all bids on every lot for the given users"),
 			new ApiCommand("PlaceBid", args => GetAuction(args.Any() ? int.Parse(args.First()) : throw new ApiError("Auction ID was not specified")).PlaceBid(args.Skip(1).Any() ? int.Parse(args.Skip(1).First()) : throw new ApiError("User ID was not specified"), args.Skip(2).Any() ? args.Skip(2).First() : throw new ApiError("Lot ID was not specified"), args.Skip(3).Any() ? int.Parse(args.Skip(3).First()) : throw new ApiError("Amount was not specified")).Wait(), new List<ApiParameter> { new ApiParameter("Auction ID"), new ApiParameter("User ID"), new ApiParameter("Lot ID", "string"), new ApiParameter("Amount") }, "Places a bid on the specified lot"),
 			new ApiCommand("PlaceBids", args => GetAuction(args.Any() ? int.Parse(args.First()) : throw new ApiError("Auction ID was not specified")).PlaceBids(args.Skip(1).Any() ? JsonConvert.DeserializeObject<List<Tuple<int, string, int>>>(args.Skip(1).First()) : throw new ApiError("Bids were not specified")).Wait(), new List<ApiParameter> { new ApiParameter("Auction ID"), new ApiParameter("Bids", "List<Tuple<int, string, int>>")}, "Places bids for multiple users, the first item of each tuple is the User ID, the second is the Lot ID, the third is the Amount"),
-			new ApiCommand("GetWinner", args => GetAuction(args.Any() ? int.Parse(args.First()) : throw new ApiError("Auction ID was not specified")).GetWinner(args.Skip(1).Any() ? bool.Parse(args.Skip(1).First()) : false).Wait(), new List<ApiParameter>{ new ApiParameter("Discount Later bidders", "bool", true) }, "Gets the winner of the given auction")
+			new ApiCommand("GetWinner", args => GetAuction(args.Any() ? int.Parse(args.First()) : throw new ApiError("Auction ID was not specified")).GetWinner(args.Skip(1).Any() ? bool.Parse(args.Skip(1).First()) : false).Wait(), new List<ApiParameter>{ new ApiParameter("Discount Later bidders", "bool", true) }, "Gets the winner of the given auction"),
+			new ApiCommand("ReplaceBid", args =>
```

Comment:
If the API function is going to be more than a simple expression, please make it its own method.